front/app/components/admin/PageWrapper/index.tsx

```diff
@@ -1,15 +1,27 @@
-import React, { PureComponent } from 'react';
+import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 import { media } from 'utils/styleUtils';
 
-const Wrapper = styled.div`
+const Wrapper = styled.div<{ flatTopBorder: boolean }>`
   background: #fff;
-  border-radius: ${(props: any) => props.theme.borderRadius};
   border: 1px solid ${(props) => props.theme.colors.divider};
   box-sizing: border-box;
   padding: 4rem 4rem;
   margin-bottom: 0px;
 
+  ${(props) => {
```

Comment:
I believe this fix is good, I'm just wondering why we don't use the default wrapper? https://github.com/CitizenLabDotCo/citizenlab/blob/1d27c60043cd3b06040f76073425e0d599ffc739/front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/index.tsx#L79
Do you remember?

Reply:
The designs are different, that one doesn't have `border-radius` and some different padding than the admin `PageWrapper` we're using